features/schema.py

```diff
@@ -1,16 +1,24 @@
 import graphene
 import graphql_geojson
 from django.apps import apps
-from graphene import relay
+from graphene import ObjectType, relay
 from graphene_django import DjangoConnectionField, DjangoObjectType
 
 from features import models
 
 
 class Url(graphene.String):
-    """
-    The `Url` scalar type represents a reference to a web resource.
-    """
+    """The `Url` scalar type represents a reference to a web resource."""
```

Comment:
would be good to add some parsing/serializing logic to this scalar to raise ValueErrors, when the format does not match the url format. Maybe not now, but at some point. 🙂

Reply:
similar to [this](https://github.com/graphql-python/graphene/blob/master/graphene/types/scalars.py#L74), for example

Reply:
I was going to just use `String` for the urls, but our initial specification said the scalar type should be `Url`. Not sure if that's good reasoning for adding a new type at since I didn't add any extra logic related to the scalar type. :man_shrugging: The initial idea (for me) was that parsing/serializing logic could be added later, perhaps at the point when we start using it for inputs.

Reply:
I decided that maybe urls can be `String`s for now, and `Url`s once it makes more sense to add the parsing/serializing logic for the new scalar type.